src/wasm-ir-builder.h

```diff
@@ -220,10 +220,26 @@ class IRBuilder : public UnifiedExpressionVisitor<IRBuilder, Result<>> {
   [[nodiscard]] Result<> visitStructNew(StructNew*);
   [[nodiscard]] Result<> visitArrayNew(ArrayNew*);
   [[nodiscard]] Result<> visitArrayNewFixed(ArrayNewFixed*);
+  // Used to visit break exprs when traversing the module in the fully nested
+  // format. Break label destinations are assumed to have already been visited,
+  // with a corresponding push onto the scope stack. As a result, an exception
+  // will occur if a corresponding scope is not found for the break.
   [[nodiscard]] Result<> visitBreak(Break*,
                                     std::optional<Index> label = std::nullopt);
+  // Used to visit break nodes when traversing the module in the stacky format.
+  // The type indicates how many values need to be popped from the stack and
+  // consumed by the break.
```

Comment:
It's important that the condition (if any), which is also popped from the stack, is _not_ included in this.
```suggestion
  // The type indicates how many values the break carries to its destination.
```
Similar comments apply to `visitSwitchWithType` below.

Reply:
done